assets/externalized/mercs-profiles.json

```diff
@@ -0,0 +1,12326 @@
+[
+    {
+        "profile": "BARRY",        // Required. Must correspond to the internalName field in mercs-profile-info.json
```

Comment:
Should we merge `mercs-profile-info.json` into `mercs-profiles.json`? 

There is no point to have two separate JSONs and hence the need to be consistent. Both map to same data classes, and the `mercs-profile-info` is just a single data field.

Reply:
I'm not sure about the mod compatibility. Are there even enough of them to care? Or are they all primarily weapons-focused?

Reply:
I guess if we almost didn't have any char externalization why would anyone bother modding it? I'll merge it then.

Reply:
How about a simple document to log all mod-breaking changes? That would help people to change customized game settings or mods.